lib/msal-node/docs/initialize-public-client-application.md

```diff
@@ -0,0 +1,65 @@
+# Initialization of MSAL
+
+Before you get started, please ensure you have completed all the [prerequisites](../README.md#prerequisites).
+
+In this document:
+- [Initializing the PublicClientApplication object](#initializing-the-publicclientapplication-object)
+- [Configuration](#configuration_basics)
+- [(Optional) Configure Authority](#optional-configure-authority)
+- [(Optional) Advanced Configuration](#advanced-configuration)
+
+## Initializing the PublicClientApplication object
+
+In order to use MSAL.js, you need to instantiate a [PublicClientApplication](https://azuread.github.io/microsoft-authentication-library-for-js/ref/msal-node/classes/_src_client_publicclientapplication_.publicclientapplication.html) object.
+
+```javascript
+import * as msal from "@azure/msal-node";
+
+const publicClientConfig = {
+    auth: {
+        clientId: "your_client_id",
+        authority: "your_authority",
+        // mandatory only for authorization code flow
+        redirectUri: "your_redirect_uri",
```

Comment:
I think node doesn't have redirectUri as part of public client config. It's only there for the auth code requests.

Reply:
You are right. Done.